config/initializers/sentry.rb

```diff
@@ -0,0 +1,3 @@
+Sentry.init do |config|
+  config.dsn = ENV["SENTRY_DSN"]
+end
```

Comment:
```
W, [2021-11-14T06:18:40.307741 #1]  WARN -- sentry: ** [Sentry] sentry-rails can't detect Rails.logger. it may be caused by misplacement of the SDK initialization code
please make sure you place the Sentry.init block under the `config/initializers` folder, e.g. `config/initializers/sentry.rb`
```

これが出てきたので移した